### ISSUE TITLE
observableTransformation of Measurement data for Becker, Borghans & Blasi case

### ISSUE THREAD

**Issue by fbergmann** [ICB-DCM/PEtab#145]
I believe the observableTransformation of the measurement files of `Becker_Science2010`, `Borghans_BiophysChem1997` and `Blasi_CellSystems2016` should be set to `lin`.

**Comment by dweindl** [comment]
Check after #154

**Comment by dweindl** [comment]
@LeonardSchmiester : Could you shortly confirm that those are now correct, or forward to who is responsible for those models?

**Comment by fbergmann** [comment]
@dweindl i think we decided that the observableTransformation shouldn't make any influence, thus we fixed the ellowitz case and so this issue is no longer needed and can be closed

**Comment by dweindl** [comment]
Thanks. Closing.
